grpc_client.go

```diff
@@ -3,26 +3,23 @@ package chalk
 import (
 	"context"
 	aggregatev1 "github.com/chalk-ai/chalk-go/gen/chalk/aggregate/v1"
-	commonv1 "github.com/chalk-ai/chalk-go/gen/chalk/common/v1"
 	"time"
 )
 
 // GRPCClient is the gRPC-native interface for interacting with Chalk.
```

Comment:
```suggestion
// GRPCClient is the gRPC-native interface for interacting with Chalk.
```
this isn't true anymore right?

Reply:
`Our existing Client interface also works with gRPC, but`

Reply:
ah thank you! nice catch